docs/source/tags/shortcut.md

```diff
@@ -12,11 +12,12 @@ Use with the following data types: audio, image, HTML, paragraphs, text, time se
 
 ### Parameters
 
-| Param | Type | Description |
-| --- | --- | --- |
-| value | <code>string</code> | The value of the shortcut |
-| [alias] | <code>string</code> | Shortcut alias |
-| [hotkey] | <code>string</code> | Hotkey |
+| Param | Type | Default | Description |
+| --- | --- | --- | --- |
+| value | <code>string</code> |  | The value of the shortcut |
```

Comment:
Should value be `[value]` instead of `value`?

Reply:
it's required